collections/_article/dev-snapshot-godot-4-1-dev-1.md

```diff
@@ -0,0 +1,131 @@
+- Physics: Add `get_contact_local_velocity_at_position` to PhysicsDirectBodyState2D ([GH-76051](https://github.com/godotengine/godot/pull/76051)).
+- Porting: Fix clipboard relying on focused window ([GH-73878](https://github.com/godotengine/godot/pull/73878)).
+- Porting: Fix queuing utterances in rapid succession in Windows TTS ([GH-75880](https://github.com/godotengine/godot/pull/75880)).
+- Porting: Fix the sliding window problem on Linux ([GH-76040](https://github.com/godotengine/godot/pull/76040)).
+- Rendering: Add EXPOSURE built in to spatial shaders ([GH-71364](https://github.com/godotengine/godot/pull/71364)).
+- Rendering: Fix for OccluderPolygon2D memory leak ([GH-74891](https://github.com/godotengine/godot/pull/74891)).
+- Rendering: Fix the limit for interpolation with respect to metallic and calculations in the SSR Fresnel Shlick ([GH-75368](https://github.com/godotengine/godot/pull/75368)).
+- Rendering: Use MODELVIEW_MATRIX when on double precision ([GH-75462](https://github.com/godotengine/godot/pull/75462)).
+- Rendering: Move sky luminance scaling to before fog is applied ([GH-75812](https://github.com/godotengine/godot/pull/75812)).
+- Rendering: Clamp normal when calculating 2D lighting to avoid artifacts ([GH-76240](https://github.com/godotengine/godot/pull/76240)).
+- Shaders: Fix crashes caused due to missing type specifier on visual shader editor ([GH-75809](https://github.com/godotengine/godot/pull/75809)).
+- Shaders: Fix completion of `source_color` hint for texture arrays in shaders ([GH-75831](https://github.com/godotengine/godot/pull/75831)).
+- XR: Add support for getting and setting display refresh rate in WebXR ([GH-72938](https://github.com/godotengine/godot/pull/72938)).
+- XR: Add a get_system_info method to XRInterface ([GH-74848](https://github.com/godotengine/godot/pull/74848)).
+- Thirdparty: HarfBuzz 7.1.0, thorvg 0.8.4, mbedtls 2.28.3
+- Documentation and translation updates.
+
+This release is built from commit [db1302637](https://github.com/godotengine/godot/commit/db1302637023168f7becceb1c4ce13228e1b2a43).
+
+## Downloads
+
+The downloads for this dev snapshot can be found directly on our repository:
+
+* [Standard build](https://downloads.tuxfamily.org/godotengine/4.1/dev1/) (GDScript, GDExtension).
+* [.NET 6 build](https://downloads.tuxfamily.org/godotengine/4.1/dev1/mono) (C#, GDScript, GDExtension).
+  - Requires [.NET SDK 6.0](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) or [7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) installed in a standard location.
+
+## Bug reports
+
+As a tester, you are encouraged to [open bug reports](https://github.com/godotengine/godot/issues) if you experience issues with this release. Please check the [existing issues on GitHub](https://github.com/godotengine/godot/issues) first, using the search function with relevant keywords, to ensure that the bug you experience is not already known.
```

Comment:
```suggestion
As a tester, we encourage you to [open bug reports](https://github.com/godotengine/godot/issues) if you experience issues with this release. Please check the [existing issues on GitHub](https://github.com/godotengine/godot/issues) first, using the search function with relevant keywords, to ensure that the bug you experience is not already known.
```

Reply:
This is a standard block, FYI. We have it in every post.